templates/src/app.js

```diff
@@ -27,8 +31,25 @@ app.use("/auth", authRoutes);<% end %>
 
 const port = process.env.SERVER_PORT || 8080;
 
+<%if ne (index .Params `cacheStore`) "none" %>
+function touchCacheStore() {
+  // memcached example getting and setting key value
+  cacheStore.set('foo', 'bar', 10,
+    function (err) {
+      if (err) throw new Error(err);
+      console.log("Set operation successful");
+    });
+  cacheStore.get('foo',
+    function (err, data) {
+      if (err) throw new Error(err);
+      console.log(`Get operation successful, key: ${data}`)
+    });
+};
+<% end %>
+
 const main = async () => {
   // remove this block for development, just for verifying DB
+  <% if ne (index .Params `cacheStore`) "none" %>touchCacheStore();<% end %>
```

Comment:
Changing the function name fixes the agnostic problem. [Redis](https://www.npmjs.com/package/redis) provides a similar API so the Memcached and Redis can both use this function.